pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,133 @@
+{ config
+, fetchurl
+, atspiSupport ? true, at_spi2_core ? null
+, gtk3
+, gnome3
+, libcanberra_gtk3
+, libudev
+, hunspell
+, isocodes
+, pkgconfig
+, xorg
+, libxkbcommon
+, python3
+, stdenv
+, bash
+, intltool
+, glib
+, gobjectIntrospection
+, gsettings_desktop_schemas
+, wrapGAppsHook
+, yelp
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  name = "onboard-${version}";
+  majorVersion = "1.4";
+  version = "${majorVersion}.1";
+  src = fetchurl {
+    url = "https://launchpad.net/onboard/${majorVersion}/${version}/+download/${name}.tar.gz";
+    sha256 = "01cae1ac5b1ef1ab985bd2d2d79ded6fc99ee04b1535cc1bb191e43a231a3865";
+  };
+
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    gtk3
+    libcanberra_gtk3
+    libudev
+    bash
+    hunspell
+    isocodes
+    pkgconfig
+    xorg.libXtst
+    xorg.libxkbfile
+    libxkbcommon
+    intltool
+    python3
+    python3.pkgs.pycairo
+    python3.pkgs.dbus-python
+    python3.pkgs.pygobject3
+    python3.pkgs.systemd
+    python3.pkgs.distutils_extra
+    python3.pkgs.pyatspi
+    glib
+  ];
+
+  buildInputs = [
+    glib gobjectIntrospection gsettings_desktop_schemas gnome3.dconf wrapGAppsHook
+  ] ++ stdenv.lib.optional atspiSupport at_spi2_core;
+
+  preBuild = ''
+    rm -r Onboard/pypredict/attic
```

Comment:
Could you add comment why this is necessary?

Reply:
I see now. It works fine for me with the directory not removed but given the reasoning above, removing it is fine. Link to https://github.com/NixOS/nixpkgs/pull/24986#issuecomment-296114062 would be enough.